lib/index.js

```diff
@@ -125,6 +125,10 @@ class CryptoRpcProvider {
   getAccountInfo(params) {
     return this.get(params.currency).getAccountInfo(params);
   }
+
+  stop(params) {
+    return this.get(params.currency).stop();
```

Comment:
would want this method to exist on other implementations, at least as a stub

Reply:
Finally addressed this